scripts/validate.py

```diff
@@ -51,6 +52,11 @@ def run(global_config):
             "Expecting non-empty IGNORED_FILES. Please check if ignored_files_manifest.csv is configured correctly!!!"
         )
 
+    logger.info(
+        "If validation job is run with FORCE_CREATE_MANIFEST True: errors from missing objects are to be expected due to redaction of records in the data release following the current run"
+    )
+    force_create_manifest = global_config.get("force_create_manifest")
```

Comment:
hah you're gonna hate me. Can you change these to all caps since its a global variable :grimacing:

Reply:
all of those var from the config?